geomesa-filter/src/test/scala/org/locationtech/geomesa/filter/function/FastPropertyTest.scala

```diff
@@ -49,7 +49,7 @@ class FastPropertyTest extends Specification {
       val sf = new ScalaSimpleFeature("id", sft)
       sf.setAttributes(Array[AnyRef]("myname", "POINT(45 45)"))
 
-      val filter = ECQL.toFilter("testns:name = 'myname'", FastFilterFactory.factory)
+      val filter = FastFilterFactory.toFilter(sft, "testns:name = 'myname'")
```

Comment:
if you use plain `ECQL.toFilter` you get the normal filters, instead of our optimized ones. Is that what you mean? I think the namespaces here are at the attribute level, not the feature level, so you'd have to explicitly create them that way. for instance, in the JoinProcess we sometimes use namespaces on attribute names coming back. but in general I don't think they are normally present on the persisted features, even in geoserver.